Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+
+
+namespace BootCamp.Chapter
+{    public class Lesson3
+    {   public static void Demo()
```

Comment:
I don't think you wanted it to be like that? Either way, please change the format to fit to the normal styling.

Reply:
not sure how that happened.  fixed.